travis tests passed, though getting this error after the tests:
is `coveralls` down?
```
> TheForemanDevDeps@1.22.0 coveralls /home/travis/build/theforeman/foreman
> cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
/home/travis/build/theforeman/foreman/node_modules/coveralls/bin/coveralls.js:18
        throw err;
        ^
Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! TheForemanDevDeps@1.22.0 coveralls: `cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the TheForemanDevDeps@1.22.0 coveralls script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-02-18T20_46_24_546Z-debug.log
The command "./script/travis_run_js_tests.sh" exited with 1.
```

--- a/webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/AutoComplete.test.js
+++ b/webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/AutoComplete.test.js
@@ -51,8 +51,24 @@ describe('AutoComplete', () => {
       instance.windowKeyPressHandler({
         charCode: KEYCODES.FWD_SLASH,
         preventDefault: noop,
+        target: { tagName: 'BODY' },
       });
       expect(typeahead.focus.mock.calls).toHaveLength(1);
+    });
+
+    it('pressing "forward-slash" inside an input should not trigger focus', () => {
+      const props = getProps();
+      const component = mount(<AutoComplete {...props} />);
+      const instance = component.instance();
+      const typeahead = instance._typeahead.current.getInstance();
+      typeahead.focus = jest.fn();
+      expect(typeahead.focus.mock.calls).toHaveLength(0);
+      instance.windowKeyPressHandler({
+        charCode: KEYCODES.FWD_SLASH,
+        preventDefault: noop,
+        target: { tagName: 'INPUT' },
+      });
+      expect(typeahead.focus.mock.calls).toHaveLength(0);
     });
 
     it('pressing "ESC" should trigger blur', () => {

--- a/webpack/assets/javascripts/react_app/components/AutoComplete/index.js
+++ b/webpack/assets/javascripts/react_app/components/AutoComplete/index.js
@@ -39,20 +39,34 @@ class AutoComplete extends React.Component {
   }
 
   windowKeyPressHandler(e) {
-    if (!this.props.useKeyShortcuts) {
+    const { useKeyShortcuts, handleSearch } = this.props;
+    const instance = this._typeahead.current.getInstance();
+    const { ENTER, FWD_SLASH, BACK_SLASH } = KEYCODES;
+    const didEventCameFromInput = e.target.tagName === 'INPUT';
+
+    /**
+     Disable this functionality if the event came from an input,
+     or if the 'useKeyShortcuts' is falsy.
+    */
+    if (didEventCameFromInput || !useKeyShortcuts) {
       return;
     }
-    const instance = this._typeahead.current.getInstance();
+
     switch (e.charCode) {
-      case KEYCODES.ENTER: {
-        this.props.handleSearch();
+      case ENTER: {
+        handleSearch();
         break;
       }
-      case KEYCODES.FWD_SLASH:
-      case KEYCODES.BACK_SLASH: {
-        if (!instance.state.showMenu) {
+      case FWD_SLASH:
+      case BACK_SLASH: {
+        const {
+          focus,
+          state: { showMenu },
+        } = instance;
+        const isMenuHidden = !showMenu;
+        if (isMenuHidden) {
           e.preventDefault();
-          instance.focus();
+          focus();
         }
         break;
       }

